de

```diff
@@ -0,0 +1,80 @@
+[1mindex 6009697..9a0c786 100644[m
+[1m--- a/src/config/config.json[m
+[1m+++ b/src/config/config.json[m
+[36m@@ -1,4 +1,4 @@[m
+ {[m
+[31m-    "input": "console",[m
+[32m+[m[32m    "input": "buttons",[m
+     "output": "console"[m
+ }[m
+\ No newline at end of file[m
+[1mdiff --git a/src/input/buttons.py b/src/input/buttons.py[m
+[1mindex 930ee47..c55f580 100644[m
+[1m--- a/src/input/buttons.py[m
+[1m+++ b/src/input/buttons.py[m
+[36m@@ -1,9 +1,59 @@[m
+[32m+[m[32mimport RPi.GPIO as GPIO[m
+[32m+[m[32mimport time[m
+[32m+[m
+ class ButtonsInput:[m
+[32m+[m
+[32m+[m[32m    btn_1 = 21[m
+[32m+[m[32m    btn_2 = 20[m
+[32m+[m[32m    btn_3 = 16[m
+[32m+[m[32m    btn_4 = 26[m
+[32m+[m[32m    btn_5 = 19[m
+[32m+[m[32m    btn_6 = 13[m
+[32m+[m
+[32m+[m[32m    _MOVE_RIGHT = "right"[m
+[32m+[m[32m    _MOVE_LEFT = "left"[m
+[32m+[m[32m    _ENTER = "enter"[m
+[32m+[m[32m    _BACK = "back"[m
+[32m+[m[32m    _ENTER = "enter"[m
+[32m+[m[32m    _POINT_1 = 1[m
+[32m+[m[32m    _POINT_2 = 2[m
+[32m+[m
+[32m+[m[32m    sleep_time = 0.5[m
+[32m+[m
+     def __init__(self):[m
+[31m-    	pass[m
+[32m+[m[41m    [m	[32mself._configure_raspberry_pins()[m
+ [m
+     def get_input(self):[m
+[31m-    	pass[m
+[32m+[m[41m    [m	[32mif GPIO.input(self.btn_1) == False:[m[41m [m
+[32m+[m[32m            return self._MOVE_RIGHT[m
+[32m+[m
+[32m+[m[32m        if GPIO.input(self.btn_2) == False:[m[41m [m
+[32m+[m[32m            return self._MOVE_LEFT[m
+[32m+[m
+[32m+[m[32m        if GPIO.input(self.btn_3) == False:[m[41m [m
+[32m+[m[32m            return self._ENTER[m
+[32m+[m[41m    [m
+[32m+[m[32m        if GPIO.input(self.btn_4) == False:[m[41m [m
+[32m+[m[32m            return self._BACK[m
+[32m+[m[41m    [m
+[32m+[m[32m        if GPIO.input(self.btn_5) == False:[m[41m [m
+[32m+[m[32m            return self._POINT_1[m
+[32m+[m[41m    [m
+[32m+[m[32m        if GPIO.input(self.btn_6) == False:[m[41m [m
+[32m+[m[32m            return self._POINT_2[m
+[32m+[m[41m    [m
+[32m+[m[32m        time.sleep(self.sleep_time)[m[41m [m
+ [m
+     def __del__(self):[m
+[31m-    	pass[m
+\ No newline at end of file[m
+[32m+[m[41m    [m	[32mpass[m
+[32m+[m
+[32m+[m[32m    def _configure_raspberry_pins(self):[m
+[32m+[m[32m        GPIO.setmode(GPIO.BCM)[m[41m [m
+[32m+[m[32m        GPIO.setwarnings(False)[m
+[32m+[m
+[32m+[m[32m        GPIO.setup(self.btn_1, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_2, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_3, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_4, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_5, GPIO.IN, GPIO.PUD_UP)[m
+[32m+[m[32m        GPIO.setup(self.btn_6, GPIO.IN, GPIO.PUD_UP)[m
+\ No newline at end of file[m
```

Comment:
Is this file is necessary for the project? I mean, it seems like a result of a build.